tests/config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = [{
+	deviceType: `raspberrypi3`,
+	suite: `${__dirname}/../suites/os`,
+	config: {
+		networkWired: false,
+		networkWireless: true,
+		downloadType: 'local',
+		interactiveTests: false, // redundant
+		balenaApiKey: process.env.API_KEY,
```

Comment:
This should be `process.env.BALENA_CLOUD_API_KEY` as thats the variable name for the API key in the jenkins environment

Reply:
Fixup pushed.